runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -34,6 +34,12 @@
  */
 public interface SparkPipelineOptions extends SparkCommonPipelineOptions {
 
+  @Description("The directory to save Spark History Server logs")
+  @Default.String("/tmp/spark-events/")
```

Comment:
Should we set `spark.eventLog.dir` in the Spark conf? Or does that not matter? https://github.com/apache/beam/blob/d1c8c241d57293b1fe9baa39f3b8e807b3a45d3a/runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkContextFactory.java#L88

Reply:
It doesn't matter, however for consistency I think it would be good to configure it in such a way.